src/main/java/com/programmers/springbootbasic/domain/customer/service/CustomerService.java

```diff
@@ -0,0 +1,74 @@
+package com.programmers.springbootbasic.domain.customer.service;
+
+import com.programmers.springbootbasic.common.utils.UUIDValueStrategy;
+import com.programmers.springbootbasic.domain.customer.dto.CustomerRequestDto;
+import com.programmers.springbootbasic.domain.customer.entity.Customer;
+import com.programmers.springbootbasic.domain.customer.exception.ErrorMsg;
+import com.programmers.springbootbasic.domain.customer.repository.CustomerRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.dao.DataAccessException;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+@Transactional(rollbackFor = {DataAccessException.class, IllegalArgumentException.class, RuntimeException.class})
```

Comment:
```suggestion
@Transactional(rollbackFor = {DataAccessException.class, IllegalArgumentException.class, RuntimeException.class})
```

앞에 두개 는 모두 RuntimeException 하위에 상속받은 구현체로 보여지는데 명시하신 이유가 있으실까요?

Reply:
트랙잭션이 실행됬을때 발생하는 예외들을 구분하여 명시함으로써 이러한 예외들에 의해서 RollBack됨을 확인할 수 있게 따로 명시하였는데, 생각해보니 상위에서 각 예외별로 처리를 구분하여 진행하고 있으니 불필요한 명시인 것 같습니다. 수정하도록 하겠습니다!